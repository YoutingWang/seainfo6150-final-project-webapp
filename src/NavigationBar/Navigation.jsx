import React from 'react';
import styles from './Navigation.module.css';
import Logo from "../images/Logo.jpg";
import { useHistory } from "react-router-dom";

const Navigation = (props) => {
    const history = useHistory();

    function handleClick() {
        history.push("/login");
    }

    function handleClickSign() {
        history.push("/signup");
    }

    return (
        <div className = {styles.header}>
                <div className = {styles.left} >
                    <img className = {styles.logo} src={Logo} alt="Logo" />
                    <h1 className = {styles.title}>Yummy Restaurants</h1>    
                </div>
                <div className = {styles.right}>
                    <button type="button" onClick={handleClick}>
                        Log in
                    </button>
                    <button type="button" onClick={handleClickSign}>
                        Sign up
                    </button>
                </div>

        </div>
    )
    
}

export default Navigation