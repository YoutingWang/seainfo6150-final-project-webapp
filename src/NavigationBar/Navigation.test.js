import React from "react";
import {
    render
} from "@testing-library/react";
import Navigation from "./Navigation.jsx";

it("renders the home component correctly", () => {
    const {
        container
    } = render( < Navigation / > );
    expect(container).toMatchSnapshot();
});