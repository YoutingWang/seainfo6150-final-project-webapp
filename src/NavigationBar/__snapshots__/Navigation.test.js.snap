// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the home component correctly 1`] = `
<div>
  <div
    class="header"
  >
    <div
      class="left"
    >
      <img
        alt="Logo"
        class="logo"
        src="Logo.jpg"
      />
      <h1
        class="title"
      >
        Yummy Restaurants
      </h1>
    </div>
    <div
      class="right"
    >
      <button
        type="button"
      >
        Log in
      </button>
      <button
        type="button"
      >
        Sign up
      </button>
    </div>
  </div>
</div>
`;
